jax/_src/numpy/fft.py

```diff
@@ -109,10 +109,76 @@ def _fft_core(func_name: str, fft_type: xla_client.FftType, a: ArrayLike,
   return transformed
 
 
-@implements(np.fft.fftn)
 def fftn(a: ArrayLike, s: Shape | None = None,
          axes: Sequence[int] | None = None,
          norm: str | None = None) -> Array:
+  r"""Compute the multidimensional discrete fourier transform.
+
+  JAX implementation of :func:`numpy.fft.fftn`.
+
+  Args:
+    a: input array
+    s: integer or sequence of integers. Specifies the shape of the result. If not
```

Comment:
I think a sequence is required here?

```suggestion
    s: sequence of integers. Specifies the shape of the result. If not
```

Reply:
Done